### PR TITLE
Update docs to align with main documentation and fix issue with org ID and resolvable fields

### PR DESCRIPTION
**Pull request by tothmano**
Depends on https://github.com/axiomhq/axiom-co/pull/1017
DO NOT MERGE BEFORE https://github.com/axiomhq/axiom-co/pull/1017

### CODE REVIEW

File: example/main.tf [hunk 1]
```diff
@@ -9,53 +9,60 @@ terraform {
...
 }
 
 resource "axiom_user" "test_user" {
-  name  = "axiom user"
-  email = "axiomusers@example.com"
+  name  = "test_user"
+  email = "test@abc.com"
   role  = "user"
-}
+}
```
Comment (Rambatino) [inline example/main.tf:68]: Make sure to always have trailling file line breaks on

File: example/main.tf [hunk 2]
```diff
@@ -9,53 +9,60 @@ terraform {
...
   operator         = "Above"
   range_minutes    = 5
   threshold        = 1
   notifier_ids = [
-    axiom_notifier.slack_test.id
+    axiom_notifier.test_slack_notifier.id
   ]
   alert_on_no_data = false
   notify_by_group  = false
+  resolvable = false
```
[the remark this answers is not in the document]
Comment (Rambatino) [reply example/main.tf:61]: Also, feel free to add more accpetance tests, this library could really do with having more
Especially matching the example doc ones

File: README.md [hunk 3]
```diff
@@ -13,7 +13,9 @@ Axiom lets you make the most of your event data without compromises: all your da
 ## Prerequisites
 
 - [Sign up for a free Axiom account](https://app.axiom.co/register). All you need is an email address.
-- [Create an API token in Axiom with permissions to query and ingest data](https://axiom.co/docs/reference/settings#access-overview).
+- [Create an advanced API token in Axiom](https://axiom.co/docs/reference/tokens#create-advanced-api-token) with the permissions to perform the actions you want to use Terraform for. For example, to use Terraform to create and update datasets, create the advanced API token with these permissions.
```
Comment (Rambatino) [inline README.md:16]:
```suggestion
- [Create an advanced API token in Axiom](https://axiom.co/docs/reference/tokens#create-advanced-api-token) with the permissions to perform the actions you want to use. For example, to use Terraform to create and update datasets, create the advanced API token with these permissions.
```

File: README.md [hunk 4]
```diff
@@ -13,7 +13,9 @@ Axiom lets you make the most of your event data without compromises: all your da
...
+- [Create a Terraform account](https://app.terraform.io/signup/account).
```
Comment (Rambatino) [inline README.md:17]: Why is this included?

Comment (tothmano) [reply README.md]: @Rambatino Thanks a lot for your review! I accept the suggestions. I would like to keep the prerequisite about a Terraform account. Can this procedure function without it?

Comment (Rambatino) [reply README.md:17]: Yeah I've never signed up and I don't think we should suggest that people need to sign up to a service unless they have to (unless something has changed, but I doubt it)

Comment (tothmano) [reply README.md:17]: Didn't you have to run `terraform login` at some point? It usually stores the token in `~/.terraform.d/credentials.tfrc.json` and then you don't need to worry about it again.

**Review by toppercodes — APPROVED** [review]
